src/model/UserRegistry.java

```diff
@@ -0,0 +1,26 @@
+package model;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class UserRegistry {
+
+  private static Set<User> userRegistry = new HashSet<>();
+
+  public static void addUser(User user) {
+    userRegistry.add(user);
+  }
+
+  public static Set<User> getUserRegistry() { return userRegistry; }
+
+  public static User fromName(String userName) {
+    User user = null;
+    for(User u: userRegistry) {
```

Comment:
with hashmap, you would not have to do this. You could have just done a map lookup